specification/cognitiveservices/data-plane/TextAnalytics/v2.0/TextAnalytics.json

```diff
@@ -0,0 +1,548 @@
+{
+ "swagger": "2.0",
+ "info": {
+  "version": "v2.0",
+  "contact": {
+   "name": "Microsoft Cognitive Services",
+   "url": "https://azure.microsoft.com/en-us/services/cognitive-services/text-analytics/",
+   "email": "mlapi@microsoft.com"
+  },
+  "title": "Text Analytics API",
+  "description": "The Text Analytics API is a suite of text analytics web services built with best-in-class Microsoft machine learning algorithms. \r\nThe API can be used to analyze unstructured text for tasks such as sentiment analysis, key phrase extraction and language detection. \r\nNo training data is needed to use this API; just bring your text data. \r\nThis API uses advanced natural language processing techniques to deliver best in class predictions.\r\n\r\nFurther documentation can be found in https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview"
+ },
+ "x-ms-parameterized-host": {
+  "hostTemplate": "{azureRegion}.api.cognitive.microsoft.com",
+  "parameters": [
+   {
+    "$ref": "#/parameters/AzureRegion"
+   }
+  ]
+ },
+ "securityDefinitions": {
+  "apim_key": {
+   "type": "apiKey",
+   "name": "Ocp-Apim-Subscription-Key",
+   "in": "header"
+  }
+ },
+ "basePath": "/text/analytics",
+ "schemes": [
+  "https"
+ ],
+ "paths": {
+  "/v2.0/keyPhrases": {
+   "post": {
+    "tags": [
+     "Key Phrases"
+    ],
+    "summary": "The API returns a list of strings denoting the key talking points in the input text. \r\n            We employ techniques from Microsoft Office's sophisticated Natural Language Processing toolkit. \r\n            See the <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview#supported-languages\">Text Analytics Documentation</a> for details about the languages that are supported by key phrase extraction.",
```

Comment:
Please remove the platform specific line endings - if you have more than one sentence, consider separating them into "summary" and "description".